src/components/Admin/ListSkills/ListSkills.js

```diff
@@ -363,7 +363,7 @@ class ListSkills extends React.Component {
       <FlatButton
         key={1}
         label="Change"
-        primary={true}
+        labelStyle={{ color: '#4285f4' }}
```

Comment:
The standard blue color is still not added to constants folder. Please add it as a constant and then import it here